pysatModels/utils/extract.py

```diff
@@ -627,10 +647,24 @@ def extract_modelled_observations(inst, model, inst_name, mod_name,
         raise ValueError("".join(["unknown model name for datetime: ",
                                   mod_datetime_name]))
 
-    tm_sec = (mod_datetime[1:] - mod_datetime[:-1]).min()
-    tm_sec /= np.timedelta64(1, 's')
-    ti_sec = (inst.index[1:] - inst.index[:-1]).min().total_seconds()
-    min_del = tm_sec if tm_sec < ti_sec else ti_sec
+    # Determine the appropriate time difference in seconds from the instrument
+    # and model data.  If only one time value is present, assume anything is
+    # close enough
+    dtime = mod_datetime[1:] - mod_datetime[:-1]
+    if len(dtime) == 0:
+        tm_sec = np.inf
+    else:
+        tm_sec = dmod_time.min()
+        tm_sec /= np.timedelta64(1, 's')
+
+    dtime = inst.index[1:] - inst.index[:-1]
```

Comment:
I've been tripped up with pandas formats a few times where it auto aligns things by time before doing the op and I end up with a bunch of zeros. I may be paranoid, but when taking deltas I tend to toss a .values in there to get back to a numpy array.